templates/orgs_ext/org_home.haml

```diff
@@ -85,6 +91,12 @@
                 [[ partner.replies.last_month ]]
               %td
                 [[ partner.replies.total ]]
+              %td
+                [[ partner.cases.opened_this_month ]]
+              %td
+                [[ partner.cases.closed_this_month ]]
+              %td
+                [[ partner.cases.total ]]
```

Comment:
Yes, the number of columns is a possible concern. The copy is a mouthful. Happy to split this into separate or condensed columns if needed. Would like to get the backend bits done first before focussing on the UI.